wezterm-gui/src/termwindow/mod.rs

```diff
@@ -1223,19 +1223,21 @@ impl TermWindow {
                     alert: Alert::Bell,
                     pane_id,
                 } => {
-                    match self.config.audible_bell {
-                        AudibleBell::SystemBeep => {
-                            Connection::get().expect("on main thread").beep();
+                    if self.window_contains_pane(pane_id) {
```

Comment:
I prefer early-return style to avoid excessive nesting; that will also minimize the diff here in this PR!

```suggestion
                    if !self.window_contains_pane(pane_id) {
                       return Ok(());
                    }
```